### PR TITLE
Update `ListCommand` class functionality

### PR DESCRIPTION
**Pull request by wamps-jp**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListCommand.java
Comment (CJ-Lee01) [inline src/main/java/seedu/address/logic/commands/ListCommand.java:1]: Good use of the predicate class.

File: src/main/java/seedu/address/model/Model.java
Comment (CJ-Lee01) [inline src/main/java/seedu/address/model/Model.java:1]: Good use of enums.

**Review by CJ-Lee01 — APPROVED** [review]
Looks good to me.

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -6,13 +6,17 @@
...
 import seedu.address.commons.core.GuiSettings;
 import seedu.address.model.person.Contact;
+import seedu.address.model.person.Type;
 
 /**
  * The API of the Model component.
  */
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
     Predicate<Contact> PREDICATE_SHOW_ALL_CONTACTS = unused -> true;
```
Comment (wxwern) [inline src/main/java/seedu/address/model/Model.java:16]: Suggestion for consistency - makes it clear that the contact is the value that's intentionally not checked.
```suggestion
    Predicate<Contact> PREDICATE_SHOW_ALL_CONTACTS = contact -> true;
```

**Review by wxwern — APPROVED** [review]
Overall, looks alright to me

File: src/main/java/seedu/address/logic/commands/ListCommand.java [hunk 2]
```diff
@@ -14,11 +16,21 @@ public class ListCommand extends Command {
 
     public static final String MESSAGE_SUCCESS = "Listed all contacts";
```
Comment (wxwern) [inline src/main/java/seedu/address/logic/commands/ListCommand.java:17]: Small detail, but it'd be better if this success message could be dynamically determined depending on the operation. `list --org` for instance could say `Listed all organisations` instead.

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (wamps-jp) [reply src/main/java/seedu/address/model/Model.java]: Done.

File: src/main/java/seedu/address/logic/commands/ListCommand.java (hunk 2 above)
Comment (wamps-jp) [reply src/main/java/seedu/address/logic/commands/ListCommand.java]: True - it's been added.

**Pull request merged**